src/dehtml.rs

```diff
@@ -153,35 +206,36 @@ fn dehtml_endtag_cb(event: &BytesEnd, dehtml: &mut Dehtml) {
 
     match tag.as_str() {
         "style" | "script" | "title" | "pre" => {
-            dehtml.strbuilder += &dehtml.append_prefix("\n\n");
+            *dehtml.get_buf() += "\n\n";
```

Comment:
Nice to get rid of `append_prefix`, I found it really confusing everytime I read it!